internal/filtering/blocked.go

```diff
@@ -379,6 +379,20 @@ var servicesData = []svc{{
 		"||dreamcast.hk^",
 		"||hdslb.com^",
 	},
+}, {
+	name: "warnerbrosdiscovery",
+	rules: []string{
+		"||hbomax.com^",
+		"||hbonow.com^",
+		"||hbonow.com^",
+		"||hbomax-images.warnermediacdn.com^",
+		"||art-gallery-latam.api.hbo.com^",
+		"||artist.api.cdn.hbo.com^",
+		"||hbogoasia.id^",
+		"||tntgo.tv^",
+		"||discoveryplus.com^",
+		"||go.cnn.com^",
```

Comment:
Some older rule sets aren't alphabetically sorted, unfortunately, but we strive to keep rules sorted.

Reply:
done